hbase-server/src/main/java/org/apache/hadoop/hbase/master/ServerManager.java

```diff
@@ -586,6 +588,15 @@ synchronized long expireServer(final ServerName serverName, boolean force) {
     }
     moveFromOnlineToDeadServers(serverName);
 
+    // If server is in draining mode, remove corresponding znode
+    String drainingZnode = ZNodePaths.joinZNode(zkw.getZNodePaths().drainingZNode,
+      serverName.getServerName());
+    try {
+      ZKUtil.deleteNodeFailSilent(zkw, drainingZnode);
+    } catch (KeeperException e) {
+      LOG.warn("Error deleting the znode for draining server " + drainingZnode);
```

Comment:
`LOG.warn("Error deleting the znode for draining server {}", serverName.getServerName(), e)` providing stacktrace will be better?

Reply:
fixed, thx